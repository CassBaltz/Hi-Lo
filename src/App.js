import React from 'react';
import { Provider } from 'react-redux';
import store from './redux/store';
import { getNewDeck } from './redux/actions/deck';

import Main from './components/Main';

store.dispatch(getNewDeck());
window.store = store;
const App = () => {
  return (
    <div className="App">
      <Provider store={store}>
        <Main dispatch={store.dispatch} />
      </Provider>
    </div>
  );
}

export default App;
