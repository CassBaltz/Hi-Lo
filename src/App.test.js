import React from "react";
import App from "./App";
test("renders learn react link", () => {
  const app = shallow(<App />);
  expect(toJson(app)).toMatchSnapshot();
});
