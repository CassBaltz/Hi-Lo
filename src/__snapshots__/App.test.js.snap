// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="App"
>
  <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Connect(WithStyles(Main))
      dispatch={[Function]}
    />
  </Provider>
</div>
`;
